2021/08/08 13:51:58 [error] 373#373: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.0.167, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.0.3:8090"
2021/08/08 15:01:58 [error] 390#390: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.0.167, server: _, request: "GET //get_stats/12 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.0.3:8090", referrer: "http://192.168.0.3:8090/"
2021/08/08 15:01:58 [error] 393#393: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.0.167, server: _, request: "GET //get_stats/11 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.0.3:8090", referrer: "http://192.168.0.3:8090/"
2021/08/08 15:01:58 [error] 390#390: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.0.167, server: _, request: "GET //get_stats/12 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.0.3:8090", referrer: "http://192.168.0.3:8090/"
2021/08/08 15:01:58 [error] 390#390: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.0.167, server: _, request: "GET //get_stats/11 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.0.3:8090", referrer: "http://192.168.0.3:8090/"
2021/08/08 15:01:58 [error] 393#393: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.0.167, server: _, request: "GET //get_stats/11 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.0.3:8090", referrer: "http://192.168.0.3:8090/"
2021/08/08 15:01:59 [error] 393#393: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.0.167, server: _, request: "GET //get_stats/11 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.0.3:8090", referrer: "http://192.168.0.3:8090/"
2021/08/08 15:01:59 [error] 390#390: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.0.167, server: _, request: "GET //get_stats/12 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.0.3:8090", referrer: "http://192.168.0.3:8090/"
2021/08/08 15:07:42 [error] 392#392: *55 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.0.167, server: _, request: "GET //get_stats/12 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.0.3:8090", referrer: "http://192.168.0.3:8090/"
2021/08/08 15:07:42 [error] 392#392: *1 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 192.168.0.167, server: _, request: "GET //get_stats/11 HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.0.3:8090", referrer: "http://192.168.0.3:8090/"
